### PR TITLE
Fixed PlanPrinter to produce an explain plan that can be streamed

### PR DESCRIPTION
**Pull request by matriv**

### CODE REVIEW

**Comment by matriv** [comment]
@seut Plz review

File: sql/src/main/java/io/crate/planner/PlanPrinter.java [hunk 1]
```diff
@@ -42,6 +42,7 @@
 import javax.annotation.Nullable;
 import java.util.ArrayList;
 import java.util.Collection;
```
Comment (seut) [inline sql/src/main/java/io/crate/planner/PlanPrinter.java]: unused import?

Comment (matriv) [inline sql/src/main/java/io/crate/planner/PlanPrinter.java]: Nope. it's used as arg of refs method

**Comment by seut** [comment]
lgtm

**Pull request merged**